main.go

```diff
@@ -311,7 +306,7 @@ func main() {
 		dispatchers := make([]func(), 0, len(sources))
 		for _, s := range sources {
 			sid := s.GetSourceId()
-			if sid == "" || sid == controllers.PredictorCRSourceId || sid == controllers.TrainedModelCRSourceId {
```

Comment:
Ah, just noticed that we are missing the check for controllers.InferenceServiceCRSourceId here. Can you just replace
`controllers.TrainedModelCRSourceID` here with `controllers.InferenceServiceCRSourceId` ?

Reply:
Sure, changed according to comments.